modules/popup/react/stories/stories.tsx

```diff
@@ -25,7 +25,7 @@ class PopupWrapper extends React.Component<{}, PopupWrapperState> {
         <Button variant={Button.Variant.Delete} onClick={this.handleClick}>
           Delete Item
         </Button>
-        <Popper placement={'bottom'} open={open} anchorEl={anchorEl}>
+        <Popper placement={'bottom'} open={open} anchorElement={anchorEl}>
```

Comment:
I'm assuming we like `anchorElement` over `anchorEl`? Our Popper API is a little different than the material-ui one